ibind/base/rest_client.py

```diff
@@ -6,6 +6,12 @@
 import requests
 from requests import ReadTimeout, Timeout
 
+
+import OAuth.oauth_requests as oauth_requests
```

Comment:
Why do we import this?

Reply:
We can remove it, I imported it to find the bugs